ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -47,6 +47,13 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: installer-rehearse-aws-ccm-qe
```

Comment:
No need to add this new test item, just replace the above line 49 with the your workflow for the rehearsed testing.